<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="EntertainmentLabel_2" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>241</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <widget class="QLabel" name="EntertainmentLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Entertainment</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>241</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 85, 255);</string>
    </property>
    <widget class="QLabel" name="EntertainmentLabel_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Productivity</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>241</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(135, 135, 135);</string>
    </property>
    <widget class="QLabel" name="EntertainmentLabel_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Other</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="EntertainmentTable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>241</width>
      <height>411</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Names</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="ProductivityTable">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>241</width>
      <height>411</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Names</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="OtherTable">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>160</y>
      <width>241</width>
      <height>411</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Names</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
